plugins/datadog/app/models/datadog_notification.rb

```diff
@@ -20,34 +19,19 @@ def deliver(additional_tags: [], now: false)
         "error"
       end
 
-    event = Dogapi::Event.new(
-      body,
-      msg_title: @deploy.summary,
-      event_type: "deploy",
-      event_object: Digest::MD5.hexdigest("#{Time.new}|#{rand}"),
-      alert_type: status,
-      source_type_name: "samson",
-      date_happened: now ? Time.now : @deploy.updated_at,
-      tags: @stage.datadog_tags_as_array + ["deploy", *additional_tags]
-    )
-
-    client = Dogapi::Client.new(api_key, nil, "")
-    status = client.emit_event(event)[0]
-
-    if status == "202"
-      Rails.logger.info "Sent Datadog notification"
-    else
-      Rails.logger.info "Failed to send Datadog notification: #{status}"
+    response = Faraday.post "https://api.datadoghq.com/api/v1/events?api_key=#{DatadogMonitor::API_KEY}" do |request|
+      request.body = {
+        title: @deploy.summary,
+        text: "#{@deploy.user.email} deployed #{@deploy.short_reference} to #{@stage.name}",
+        alert_type: status,
+        source_type_name: "samson",
+        date_happened: now ? Time.now.to_i : @deploy.updated_at.to_i,
+        tags: @stage.datadog_tags_as_array + ["deploy", *additional_tags]
+      }.to_json
     end
-  end
 
-  private
-
-  def body
-    "#{@deploy.user.email} deployed #{@deploy.short_reference} to #{@stage.name}"
-  end
-
-  def api_key
-    ENV["DATADOG_API_KEY"]
+    if response.status != 202
```

Comment:
What about statuses
- 200 OK
- 201 Created?
https://docs.datadoghq.com/api/?lang=ruby#success-and-errors

Reply:
yeah let's just say between 200 and 202 ... 202 is what datadog returns, but no need to hardcode this weirdness